mavros/src/lib/ftf_frame_conversions.cpp

```diff
@@ -84,7 +84,7 @@ Eigen::Vector3d transform_static_frame(const Eigen::Vector3d &vec, const StaticT
 	switch (transform) {
 	case StaticTF::NED_TO_ENU:
 	case StaticTF::ENU_TO_NED:
-		return NED_ENU_AFFINE * vec;
+		return Eigen::Vector3d(vec.y(), vec.x(), -vec.z());
 
```

Comment:
Add a check for NaNs here and apply the respective return depending on the result.

Reply:
Why do extra checks? This gives the same result, but without the floating point issues.